pyronear/datasets/wildfire/wildfire.py

```diff
@@ -72,7 +75,7 @@ def _get_targets(self, index):
         return torch.from_numpy(self.metadata[self.target_names].iloc[index].values)
 
 
-class WildFireSplitter:
+class WildFireSplitter(object):
```

Comment:
Same all classes are new-style in python 3, so we no longer need to inherit from object.